src/lib/utils.js

```diff
@@ -10,6 +10,9 @@ import _ from 'lodash';
 import createError from 'http-errors';
 import type {Package} from '@verdaccio/types';
 import type {$Request} from 'express';
+import marked from 'marked';
+// $FlowFixMe
+import asciidoctor from 'asciidoctor.js';
```

Comment:
You can use here create-stub 
https://github.com/flowtype/flow-typed/wiki/CLI-Commands-and-Flags

Reply:
Or try to find the lib whether exist